add_map_plugin.py

```diff
@@ -0,0 +1,41 @@
+import sys
```

Comment:
Also, please add Apache 2.0 license header attributed to Amazon (year 2020).

Reply:
Updated license on root dir according to https://aws.amazon.com/apache-2-0/